plugins/services/src/js/structs/Framework.js

```diff
@@ -1,5 +1,5 @@
 import { cleanServiceJSON } from "#SRC/js/utils/CleanJSONUtil";
-import { isSDKService } from "#SRC/js/utils/ServiceUtil";
+import { isSDKService } from "#PLUGINS/services/src/js/utils/ServiceUtil";
```

Comment:
There are two `ServiceUtil`, the one in plugins includes `Service` which creates a circular dependency…